### PR TITLE
Mitigate flickering error screen on user eject/meeting ended

### LINKED ISSUE #7209: Removing Annotations outside Fiber
I20190409-18:18:32.967(0)? error: Removing Annotations from collection: Error: Meteor code must always run within a Fiber. Try wrapping callbacks that you pass to non-Meteor libraries with Meteor.bindEnvironment.

### PR DESCRIPTION
**Pull request by antobinary**
* fixed collections not being removed due to `Error: Meteor code must always run within a Fiber. Try wrapping callbacks that you pass to non-Meteor libraries with Meteor.bindEnvironment.` Closes #7209 

* reopens #7285

### CODE REVIEW

**Comment by capilkey** [comment]
I don't know if this is from this PR or not, but when I enable the feedback page and it's loaded I see see the following warning.
image

**Comment by capilkey** [comment]
Another thing I've noticed is that when I have feedback enabled, I'm not directed to the feedback page if the meeting is ended or if I am removed. If I logout, I do see the feedback page.

**Comment by ffdixon** [comment]
> Another thing I've noticed is that when I have feedback enabled, I'm not directed to the feedback page if the meeting is ended or if I am removed. If I logout, I do see the feedback page.

This is not a show stopper -- we can improve this in later update.

**Comment by capilkey** [comment]
The breakout room windows are now closing as expected again.

File: bigbluebutton-html5/imports/api/meetings/server/handlers/meetingEnd.js [hunk 1]
```diff
@@ -8,22 +8,24 @@ import Users from '/imports/api/users/';
 export default function handleMeetingEnd({ body }, meetingId) {
   check(meetingId, String);
 
+  const cb = () => {
+    Users.find({ meetingId }).fetch().map(user => setConnectionStatus(user.meetingId, user.userId, 'offline'));
+    return Meteor.setTimeout(() => { meetingHasEnded(meetingId); }, 1000);
```
Comment (jfsiebel) [inline bigbluebutton-html5/imports/api/meetings/server/handlers/meetingEnd.js]: Shouldn't be at least `10000` ms?

**Pull request merged**